bin/check-package-metadata.js

```diff
@@ -118,6 +122,11 @@ async function checkPkgsPackageJson(packages, rootPkg) {
       errors.push(getErrorText(p.name, 'types'));
     }
 
+    if (getEngineSemverRange(pkg) !== getEngineSemverRange(rootPkg)) {
+      console.log(getEngineSemverRange(pkg) + getEngineSemverRange(rootPkg));
+      errors.push(getErrorText(p.name, 'engines'));
```

Comment:
@achrinza Thank you for adding the check.

I would like to go one step further to have a script to automatically populate/update `node.engines` for all packages using the value in `loopback-next/package.json`. We can run the script as part of the release process.